hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/listeners/DomainEventListener.kt

```diff
@@ -60,6 +61,6 @@ data class IncentiveChangeAdditionalInformation(
   val id: Long,
 )
 
-data class EventType(val Value: String)
+data class EventType(@JsonProperty("Value") val Value: String)
```

Comment:
this was an odd one - without defining the json property it would fail due to the constructor parameter being called `Value`.  If I called it anything else it was happy 🤷